exec/vector/.externalToolBuilders/Maven_Ant_Builder.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did this file want to be committed? Looks like an Eclipse-specific launch script...